databricks-sdk-java/src/main/java/com/databricks/sdk/core/http/Request.java

```diff
@@ -40,7 +40,12 @@ public Request withHeader(String key, String value) {
   }
 
   public Request withQueryParam(String key, String value) {
-    query.put(key, value);
+    List<String> values = query.get(key);
+    if (values == null) {
+      values = new ArrayList<>();
+    }
+    values.add(value);
+    query.put(key, values);
```

Comment:
Should we handle the use case for when key exists?

Reply:
if the key exists, values is not null, so we just skip the if block. Note that I've refactored this, as the put at the end is only needed in the case that values is null.